css/tabBar.css

```diff
@@ -125,6 +124,9 @@ body.linux #menu-button {
 .tab-item.fade:hover {
 	opacity: 1;
 }
+.tab-item .title {
+	text-overflow: ellipsis;
+}
```

Comment:
Can you remove this for now? it doesn't actually work without switching the tab contents to a flexbox-based layout, and I think I'd rather use a fade-out effect like in https://github.com/minbrowser/min/pull/1109#issuecomment-653810851.

Reply:
Yeah, sorry it was just a expirement that I forget to remove.